src/i18n.js

```diff
@@ -190,7 +242,33 @@ const messages = {
     'navbar.about': '关于我们',
     'navbar.docs': '文档',
     'navbar.faq': '常见问题及解答',
-    'footer.copyright': '© 2023 EOS 网络基金会保留所有权利。'
+    'footer.copyright': '© 2023 EOS 网络基金会保留所有权利。',
+
+    'app.walletconnect.failedswitch.title': '钱包中缺失网络配置',
+    'app.walletconnect.failedswitch.content': `<p>我们检测到您尝试使用的钱包尚未配置以支持EOS EVM。请使用以下网络信息添加网络，或访问我们的<a href="https://docs.eosnetwork.com/evm/quick-start/introduction" >文档页面</a>获取更多详细信息。</p>
```

Comment:
Please use the following link for ZH https://docs.eosnetwork.com/zh/evm/quick-start/setup-metamask/

Reply:
done